nncf/quantization/algorithms/fast_bias_correction/ov_backend.py

```diff
@@ -0,0 +1,165 @@
+        return {
+            OV_OPERATION_METATYPES.get_operator_metatype_by_op_name('Convolution'): 1,
+            OV_OPERATION_METATYPES.get_operator_metatype_by_op_name('ConvolutionBackpropData'): 1,
+            OV_OPERATION_METATYPES.get_operator_metatype_by_op_name('MatMul'): -1
+        }
+
+    @property
+    def tensor_processor(self) -> OVNNCFCollectorTensorProcessor:
+        return OVNNCFCollectorTensorProcessor()
+
+    @staticmethod
+    def model_transformer(model: ov.Model) -> OVModelTransformer:
+        return OVModelTransformer(model)
+
+    @staticmethod
+    def target_point(target_type: TargetType, target_node_name: str, port_id: int) -> OVTargetPoint:
+        return OVTargetPoint(target_type, target_node_name, port_id)
+
+    @staticmethod
+    def bias_correction_command(target_point: OVTargetPoint, bias_value: np.ndarray) -> OVBiasCorrectionCommand:
+        return OVBiasCorrectionCommand(target_point, bias_value)
+
+    @staticmethod
+    def model_extraction_command(inputs: List[str], outputs: List[str]) -> OVModelExtractionCommand:
+        return OVModelExtractionCommand(inputs, outputs)
+
+    @staticmethod
+    def mean_statistic_collector(reduction_shape:
+                                 ReductionShape,
+                                 num_samples: int = None,
+                                 window_size: int = None) -> OVMeanStatisticCollector:
+        return OVMeanStatisticCollector(reduction_shape, num_samples, window_size)
+
+    @staticmethod
+    def get_input_output_names(node: NNCFNode, nncf_graph: NNCFGraph) -> Tuple[str, str]:
+        input_name = node.node_name
+        output_name = node.node_name
+
+        bias_node = OVFastBiasCorrectionAlgoBackend.get_bias_node(node, nncf_graph)
+        if bias_node is not None:
+            output_name = bias_node.node_name
+        return input_name, output_name
+
+    @staticmethod
+    def create_blob(shape: Tuple[int], data: List[float]) -> np.ndarray:
+        blob = np.zeros(shape)
+        for i, value in enumerate(data):
+            blob[:, i] = value
+        blob = blob.astype(np.float32)
+        return blob
+
+    @staticmethod
+    def get_bias_value(model: ov.Model, bias_node: NNCFNode) -> np.ndarray:
+        ops_dict = {op.get_friendly_name(): op for op in model.get_ops()}
+
+        bias_node = ops_dict[bias_node.node_name]
+        skip_metatypes = ['Convert']
+        needed_bias_type = 'Constant'
+
+        potential_bias_queue = deque([n.node for n in bias_node.input_values()])
+        while potential_bias_queue:
+            potential_bias = potential_bias_queue.popleft()
+            if potential_bias.get_type_name() in skip_metatypes:
+                # We goes thorough 0 port in assumption that bias graph without branching
+                potential_bias_queue.append(potential_bias.input_value(0).node)
+            elif potential_bias.get_type_name() == needed_bias_type:
+                bias_value = potential_bias.get_data()
```

Comment:
In case of FP16 `get_data` returns `np.uint16` array, but `get_vector` returns `np.float16`.
```suggestion
                bias_value = potential_bias.get_vector()
                return bias_value, potential_bias.get_output_shape(0)
```

Reply:
Fixed.